docs/Configuration.md

```diff
@@ -124,15 +129,20 @@ flinkConfig {
 }
 ```
 In this section you can put all configuration values for Flink client. We are using Flink REST API so the only
-required parameter is restUrl - which defines location of Flink JobManager
-* jobManagerTimeout (e.g. 1m) - timeout used in communication with Flink cluster
-* classpath - list of files/URLs with jars with model for processes 
+required parameter is `restUrl` - which defines location of Flink JobManager
+* `jobManagerTimeout` (e.g. 1m) - timeout used in communication with Flink cluster
+* `classpath` - list of files/URLs with jars with model for processes
 
 ##Process  {#model}
 
 Configuration of processes has few common keys:
-*  timeout (e.g. 10s)- for synchronous services
-*  checkpointInterval - e.g. 10m
+*  `timeout` (e.g. 10s)- for synchronous services
+*  `checkpointInterval` - deprecated (use `checkpointConfig.checkpointInterval` instead), e.g. 10m
+*  `checkpointConfig` (more about checkpoint configuration you can find in [Flink Documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.9/api/java/org/apache/flink/streaming/api/environment/CheckpointConfig.html))
+    * `checkpointInterval` - e.g. 10m
+    * `minPauseBetweenCheckpoints` - optional, default to half of `checkpointInterval`, e.g. 5m
+    * `maxConcurrentCheckpoints` - optional, default to 1, e.g. 4
+    * `tolerableCheckpointFailureNumber` - optional, default not set, e.g. 6
```

Comment:
what does 'default not set' mean in this setting? Int.Max?

Reply:
I set the default to 0 as it is in flink docs

Reply:
Let's write that in case of missing, will be used Flink's default (0)